docs/fr/getting-started.md

```diff
@@ -1,19 +1,19 @@
-# Débuter
+# Pour commencer
 
-Au cœur de chaque application Vuex, il y a le **store**. Un "store" est tout simplement un conteneur avec le **state** de votre application. Il y a deux choses qui différencient un store Vuex d'un simple objet global :
+Au cœur de chaque application Vuex, il y a la **zone de stockage (« store »)**. Un « store » est tout simplement un conteneur avec l'**état** de votre application. Il y a deux choses qui différencient un store Vuex d'un simple objet global :
```

Comment:
Merci Posva pour tes retours !

**Nous traquons l'état des traductions des termes ici : https://github.com/vuejs-fr/vuejs.org/issues/4
- Quand un terme générique existe dans la culture française nous le reprenons (ex : portée pour scope).
- Cependant quand le terme est spécifique à une techno, nous nous référons à la documentation française. Si elle n'existe pas, nous gardons le terme anglais (ex : scope pour scope) avec angular.
- Quand le terme générique n'existe pas, nous cherchons une équivalence anglaise (ex : état pour state),
- Si cela n'est pas possible, nous conservons le terme original (ex : store, props, slot).
- Si le terme traduit ne reflète pas assez ce qu'il signifiait en anglais, nous ajoutons la version originale entre parenthèse après pour lever l’ambiguïté.**

Nous avons ajouté « store » entre parenthèse car nous ne réutilisons pas le terme **zone de stockage** pour chaque itération de store faute de traduction plus probante.

En ce qui concerne les termes « État », « État local », « État global » nous envisageons dans faire les termes usuel.

De la même manière que (par exemple) le moyen de connaître la version anglaise du terme [opérateur de décomposition](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Op%C3%A9rateurs/Op%C3%A9rateur_de_d%C3%A9composition) est d'aller sur la version anglaise [Spread Operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator) nous ne souhaitons pas accoler le terme original pour chaque terme traduit sauf si nous envisageons plus loin dans la documentation de le réutiliser.

À mon sens, s'il y a quelque chose à mettre en place pour faciliter la mise en relation des termes traduit/originaux ; c'est de pouvoir permuter de version de langue depuis la page courante pour connaître les termes usuels d'un concept dans chaque langue plutôt que de forcer, pour chaque première occurrence de chaque page, la répétition original d'un concept traduit.

Aussi sachant cela, 
- Penses-tu qu'il soit toujours nécessaire de préciser « state » entre parenthèse sachant que si la documentation n'est pas lu dans le sens de lecture, l'utilisateur ne croisera pas se terme (sauf bien entendu s'il souhaite découvrir Vue côté anglais).
- Penses-tu que nous ne devrions pas traduire le terme ? 
- Penses-tu que ça « ne mange pas de pain » même en respectant notre philosophie de tout de même le traduire.

Un bon argument en faveur de laisser le terme original entre parenthèse est qu'il permet de faire le lien avec la syntaxe des exemples de code certe, mais l’exercice de traduction est également d'assumer ses choix de traduction comme MDN le fait très bien.